taxi/admin.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from taxi.models import Driver, Manufacturer, Car
+
+
+@admin.register(Driver)
+class DriverAdmin(admin.ModelAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (("Additional info",
+                                        {"fields": ("license_number",)}),)
+    add_fieldsets = UserAdmin.add_fieldsets + (("Additional info",
+                                                {"fields": (
+                                                    "firs_name",
```

Comment:
There is a typo in the field name 'firs_name'. It should be 'first_name'. This is an error related to the task requirements as it will cause issues when trying to add a new Driver through the admin interface.